docker/Dockerfile

```diff
@@ -37,11 +37,8 @@ FROM docker.io/python:${PYTHON_VERSION}-slim as requirements
 # https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md#build-mounts-run---mount.
 # Here we use it to set up a cache for apt (and below for pip), to improve
 # rebuild speeds on slow connections.
-RUN \
-   --mount=type=cache,target=/var/cache/apt,sharing=locked \
-   --mount=type=cache,target=/var/lib/apt,sharing=locked \
- apt-get update && apt-get install -y git \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y git \
```

Comment:
this seems related

Reply:
SIGH, sorry I did a `git commit -a` accidentally after fixing this up and failed to reject the diff :/